runtime/onert/core/include/backend/IExternalContext.h

```diff
@@ -1,34 +0,0 @@
-/*
- * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-#ifndef __ONERT_BACKEND_IEXTERNAL_CONTEXT_H__
-#define __ONERT_BACKEND_IEXTERNAL_CONTEXT_H__
-
-namespace onert
-{
-namespace backend
-{
-
-struct IExternalContext
-{
-  virtual ~IExternalContext() = default;
-  virtual void setMaxNumThreads(int) = 0;
```

Comment:
@periannath Is this supposed to be used by onert core?

Reply:
No, it isn't. It will be only used by backends.